sql/src/main/java/org/apache/druid/sql/calcite/schema/SegmentMetadataHolder.java

```diff
@@ -91,14 +100,14 @@ public SegmentId getSegmentId()
     return segmentId;
   }
 
-  public Map<SegmentId, Set<String>> getReplicas()
+  public Set<String> getReplicas()
   {
-    return segmentServerMap;
+    return segmentServers;
   }
 
-  public long getNumReplicas(SegmentId segmentId)
+  public long getNumReplicas()
   {
-    return segmentServerMap.get(segmentId).size();
+    return segmentServers.size();
```

Comment:
Why do we return long in this method when Set#size() returns an int?  Might not be worth changing because it might blow up other stuff.

Reply:
It used to be int, and then got changed to long at some point. The reason is because we support nulls for long [here](https://github.com/apache/incubator-druid/blob/master/sql/src/main/java/org/apache/druid/sql/calcite/table/RowSignature.java#L156), but not for ints ? Not very sure...but will likely leave it to be long here.

Reply:
Sounds good